CONTRIBUTING.md

```diff
@@ -131,3 +131,25 @@ Here's a checklist for you about what makes Pull Requests to ReSwift shine:
 * You can propose PRs to merge with the `master` branch directly. We don't use any complex branching strategies.
 
 As a contributor, choose the "squash & merge" strategy to merge PRs with a single commit, keeping the commit history clean. (That's an upside of focused Pull Requests: you don't lose extra information.)
+
+## Publishing New Releases
+
+Currently @DivineDiminion, @mjarvis, @Ben-G and @agentk are folks that have the necessary permissions to publish a new version to CocoaPods. If you want a new version of ReSwift to be published you should ping these folks.
+
+To create a new release:
+
+- [ ] Bump version number in `ReSwift.podspec` and `Info.plist` (We follow [semver](https://semver.org/) - most importantly any breaking API change should result in a major API version bump)
+
+- [ ] Add the new version number and the release date to `Changelog.md`
+
+  **Depending on release type:**
+  - [ ] If you're releasing off of `master`, simply create a PR against `master` after the version has been bumped
+  - [ ] If you're creating a release with cherry-picked fixes, create a PR against a separate release branch
+
+- [ ] Create a tag off of the relevant branch (`master` for regular release)
```

Comment:
We should specify that the tag should be created via https://github.com/ReSwift/ReSwift/releases with the changelog release notes copied over